pkg/api/handlers/fixtures/mock_sqs.go

```diff
@@ -0,0 +1,22 @@
+package fixtures
+
+import (
+	"github.com/aws/aws-sdk-go/service/sqs"
+)
+
+type MockSqs struct {
+	_ *sqs.SQS
+	_ *string
+}
```

Comment:
this can be empty you only need to provide matching functions for the interface to be in compliance